estat/README.md

```diff
@@ -0,0 +1,156 @@
+## Overview
+
+`estat` - extensible performance observability tool for Delphix Dynamic Data Platform.  
+
+This tool mainly uses `eBPF` programs with `BCC` front-end to collect and output performance statistics from kernel via `kprobes`.  
+It is made up of the following components:
+
+- `cmd/estat`
+     - Main script that provides a framework for common I/O data aggregation and drives execution of various *performance data collectors*. 
+- `lib/bcchelper.py`, `lib/bcc_helper.h`
+     - Helper script providing additional framework for data aggregation methods (scalar and histogram) and output formatting.
+- `bpf/estat/*`
+     - eBPF programs which form the *performance data collectors* (like `bpf/estat/zio.c`) using above framework.
+- `bpf/standalone/*`
+     - Standalone scripts that form additional *performance data collectors* and mostly self contained.
+
+## Installation
+
+`estat` is provided as an archive package of its components.
```

Comment:
I was looking at adding instructions on how to install this on a non-delphix ubuntu system, but realized that `estat` typically won't work on a non-delphix system, because the ZFS headers are installed in a different location. If we want this to be usable on other systems, we'll have to do some work to make that happen. Otherwise, for now, I think we can just say that this is already available on the Delphix Engine:

> `estat` is already installed on the Delphix Engine.

Reply:
Ok this sounds good, will update that line in README.md